assignments/objects.js

```diff
@@ -19,27 +19,83 @@ const example = {
 
 // Write your intern objects here:
 
+const Mitzi = {
+  "id": 0,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const Kennan = {
+  "id": 1,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const Keven = {
+  "id": 2,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const Gannie = {
+  "id": 3,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const Antonietta = {
+  "id": 4,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(Mitzi.name);
+
 // Kennan's ID
 
+console.log(Kennan.id);
+
 // Keven's email
 
+console.log(Keven.email);
+
 // Gannie's name
 
+console.log(Gannie.name);
+
 // Antonietta's Gender
 
+console.log(Antonietta.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+function speak(){
+  return "Hello, my name is " + Kennan.name + "!";
+}
+
+console.log(speak());
+
```

Comment:
This works as an independent function but not as a function of the object. You want this function to be inside of the kennan object like this <h3>
const kennan = {
  "id": 2,
  "name": "Kennan",
  "email": "kdiben1@tinypic.com",
  "gender": "M",
  speak: () => "Hello, my name is Kennan!",
}
</h3>  
Then you would call it like so: 
<h3>console.log(kennan.speak());
</h3>